### PR TITLE
Inline ga js

### LINKED ISSUE #825: Could we combine some JS files?
Would it be possible to combine some JS files?

* analytics.js
* ga.js
* main.js

Maybe roll the two into main.js?

### PR DESCRIPTION
**Pull request by cadecairos**
fixes #825 

@jbuck R?

Filed #860 to update usage of the deprecated frame-src directive

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ module.exports = function() {
         fontSrc: ['self', 'https://fonts.gstatic.com', 'https://maxcdn.bootstrapcdn.com', 'https://pontoon.mozilla.org'],
         frameSrc: ['https://js.stripe.com', 'https://checkout.stripe.com', 'https://pontoon.mozilla.org'],
         imgSrc: ['self', 'https://www.google-analytics.com', 'https://q.stripe.com', 'https://pontoon.mozilla.org'],
-        scriptSrc: ['self', 'https://cdn.optimizely.com',
+        scriptSrc: ['self', 'unsafe-inline', 'https://cdn.optimizely.com',
```
Comment (jbuck) [inline server.js]: Could you also add unsafe-eval?

File: pages/index.jsx [hunk 2]
```diff
@@ -13,6 +13,16 @@ var Index = React.createClass({
       googleFonts += "&subset=latin-ext";
     }
     var fileHashes = JSON.parse(fs.readFileSync(Path.join(__dirname, '../public/webpack-assets.json')));
+    var script = React.createElement('script')
```
Comment (jbuck) [inline pages/index.jsx]: wait really

**Comment by jbuck** [comment]
r+ :sob:

File: pages/index.jsx (hunk 2 above)
Comment (cadecairos) [inline pages/index.jsx]: cruft, removed in latest patch

**Pull request merged**